zds/tutorialv2/tests/factories.py

```diff
@@ -321,3 +322,15 @@ class Meta:
         model = ContentContributionRole
 
     title = factory.Sequence("Rôle {}".format)
+
+
+class LabelFactory(factory.django.DjangoModelFactory):
+    """Factory that create a label for publications, for use in tests."""
```

Comment:
Ce commentaire est entièrement redondant avec le code :
- c'est une factory => c'est dans le nom
- qui crée des labels => c'est dans le nom
- for use in test => oui, et on le sait parce c'est dans le module test.

Il ne faut pas toujours prendre exemple sur le reste du code, parfois, ce n'est pas optimal. ^^

Reply:
Le commentaire a été retiré.